src/test/fuzz/policy_estimator.cpp

```diff
@@ -68,10 +70,18 @@ FUZZ_TARGET(policy_estimator, .init = initialize_policy_estimator)
                     const CTransaction tx{*mtx};
                     mempool_entries.push_back(ConsumeTxMemPoolEntry(fuzzed_data_provider, tx));
                 }
-                std::vector<CTransactionRef> txs;
+                std::vector<const NewMempoolTransactionInfo> txs;
```

Comment:
863d46751900a0c69c4f7fbd92d4d8a651b2cc7b why are you making the element type const?

Reply:
This is fixed in latest push.